package.json

```diff
@@ -27,12 +27,13 @@
   },
   "dependencies": {
     "@hapi/content": "^4.1.0",
-    "dicer": "~0.3.0"
+    "dicer": "~0.3.0",
+    "event-iterator": "^1.2.0"
   },
   "devDependencies": {
     "aegir": "^20.0.0",
     "chai": "^4.2.0",
-    "ipfs-api": "github:ipfs/js-ipfs-api#1fd9749",
+    "ipfs-api": "^26.1.2",
```

Comment:
Would be nice to switch to `ipfs-http-client`...

Reply:
Done!